src/test/java/AssertJTests/TestGet.java

```diff
@@ -0,0 +1,34 @@
+package AssertJTests;
+
+import APICalls.GetCall;
+import org.testng.annotations.Test;
+import utility.PropertyManager;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestGet extends GetCall {
+
+    public TestGet() throws IOException {
+    }
+
+    @Test(priority = 0)
```

Comment:
ensure that the priority is set intentionally 
if the tests are independent, priority might not be necessary, if there is a dependency between tests, the priorities should reflect the order of execution